airbyte-webapp/src/components/JobItem/components/Logs.tsx

```diff
@@ -57,4 +61,36 @@ const Logs: React.FC<LogsProps> = ({ logsArray }) => {
   );
 };
 
+/**
+ * Matching the log's line number by time makes the following assumptions:
+ * 1. The log's lines are already ordered by time
+ * 2. The timestamps used are in the same timezone
+ */
+const getMatchingLineNumbers = (matchTimestamp: number | undefined, lines: string[] | undefined) => {
```

Comment:
This section is rather brittle, and might cause problems locking up the UI for particularly long logs.  If this is going to be a problem we could:
1. disable jumping to the right line if the log is > 10,000(?) lines
2. sample - rather than parse every line, we could parse timestamps from only 1000 lines sampled evenly.  We would jump to the closest line. 
3. pre-parse the logs to extract timestamps on the server.  I'd rather not pass this load to the server/worker, and keep this in the UI for now until we learn that this useful/utilized by our users. 

What are your thoughts?